src/main/js/controls/Graph/helpers/helpers.js

```diff
@@ -75,18 +76,45 @@ const getAxisInfoRowLabelHeight = (config) =>
  * @private
  * @param {object} config - config object derived from input JSON
  * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
- * @returns {object} d3 svg path
+ * @returns {undefined} d3 svg path
  */
-const translateDefs = (config, canvasSVG) =>
-    canvasSVG
-        .select(`clipPath#${config.clipPathId}`)
-        .selectAll("rect")
-        .attr("height", config.height)
-        .attr("width", getXAxisWidth(config))
-        .attr(
-            constants.Y_AXIS,
-            config.axis.x.orientation && calculateVerticalPadding(config)
-        );
+const translateDefs = (config, canvasSVG) => {
```

Comment:
I thought we were going to create another `clipPath` for dateline. Why are you modifying the existing one?

Reply:
Because i have to do the change at one place only.

If it's not proper we can go for different function.

Reply:
No, this is ok